### DISCUSSION TITLE
Welcome to the community!

### DISCUSSION THREAD

**Discussion by kiwicopple** [supabase/supabase#203] [General]
## 👋 Welcome!

 We use **GitHub Discussions** as a place for thoughtful, longer-form conversations — to ask non-urgent questions, share ideas, or make feature requests. It’s a space to learn from others and build collective knowledge over time.

We use **Discord** for real-time chat — to connect with other community members, get quick help, or just hang out and talk about what you’re building.

We’d love for you to join in both!

To get started here, introduce yourself below and tell us what you’re building with Supabase.
And for live conversations and quick community support, hop over to [discord.supabase.io](https://discord.supabase.io/)

**Comment by zlwaterfield** [comment]
Hey everyone, stoked to be here! Excited to contribute to Supabase and help build out this incredible project. Currently I am building a wedding guest-list management tool as a POC SaaS app. This was originally built in Firebase but converted to Supabase. Planning to launch the MVP in the next week.

**Reply by Danities316** [reply]
Please kindly share the link here once you done.....Good concept i must say

**Comment by jnutt367** [comment]
Hello friends! I am excited to be here. I have had so much trouble learning how to and trying to implement a database of my own that I thought I would try this free open source github repository that may help me out. I found it through watching a supabase overview with James Q. Quick. I will be trying to make a Full Stack Social Change Application(a social media application for ppl who believe in the need for social change). It will need a user's database and authentication and ability to scale hopefully will be necessary but getting something together that will work and show my ability to write both server side and client side languages. I feel like I have syntax heavy learning and would love to make something great. My React apps and small builds and what I have been able to accomplish is with what I know, is just not enough. I need to start learning how to leverage open source software like this so thank you in advance for any help. I promise I will give it as well as receive it.

**Comment by ali-ghn** [comment]
Hello I have an idea which I think is a big advantage over other competitors.

**Modularity**
With this feature you can make modules which contains certain database scheme, roles and etc which can simplify creating further projects.

P.S: And also I don't think you should advertise your project as "The Open Source Firebase Alternative"

**Comment by FatumaA** [comment]
Hello 👋🏼, 
My name is Fatuma and I've been a supafan since I discovered supabase in February I think. Excited to discover the community and to be here 😊

**Comment by zakaria-chahboun** [comment]
mmmm i really like this place over discord 😞

**Reply by soedirgo** [reply]
Our Discussions isn't going away 😉

**Comment by kevindmoore** [comment]
Hello there. I'm just starting on my Supabase journey. I've managed to create an instance and can create a new user. However, I'd really like to use Google Authentication but it seems to need a callback, and I'm having a really hard time to get that to work on a Mac.

**Comment by daniel-meyhoefer** [comment]
Really love supabase! 
Great docs and UX, thanks 🥰

Trying to create a small website with Svelte and supabase. Wanting to use LinkedIn auth and followed the instructions (inserting project url in linkedIn App and LinkedIn credentials in supabase). 
I can click on the LogIn button, type in my LinkedIn-credentials, allow my website to access my account and get redirected. However instead of a successful session I get the following error message in my URL bar 
`error=server_error&error_description=Unable+to+exchange+external+code%3A+AQSHg_omRSOVd33nWtE8o38xuU...`
LogIn via magic link does work however... 

Google search did not really reveal anything about that error... I used for my Svelte project the Supabase documentation skeleton
([https://supabase.com/docs/guides/with-sveltekit](url))

Many thanks in advance for help! Right now, I do not even know, where exactly I have to check for errors...

**Comment by sivalaxman07** [comment]
Hey everyone, Excited to be here and cant wait to contribute to Supabase and help build out this incredible project. 
Currently I have made a repo for the ruby client for to interact with supabase functions.
can you please review and suggest any changes or ideas.thank you🙂.
https://github.com/sivalaxman07/supabase-rubyfunc

**Comment by Danities316** [comment]
Hi Everyone.
I am here to contribute to this supa-amazing open source. I am an open source contributor beginner. Please if any one can guide me i would appreciate. Thanks

**Reply by FatumaA** [reply]
Hi and welcome,
Supabase is always happy to have contributors. 
You can find more info on that here:
https://github.com/supabase/.github/blob/f0298106d7020f8ef077bbb93ed58b3db0069a30/CONTRIBUTING.md

You can look for good first issues and docs issues. Those are good ways to get comfortable to the process and get started 🙂

**Comment by LesterToledo** [comment]
Good Afternoon Friends, I can't automate emails in Supabase using functions with Edge Function

**Reply by FatumaA** [reply]
Hey,
You could search issues to see if something similar has been raised. 
If not, consider raising one with a desc and ideally ways to reproduce.

**Comment by pluong01** [comment]
Hello All!  I am a newbie to the integration of Supabase and Flutter.  I am building a web app using this integration and have a very basic question.  Do I need another hosting server besides Supabase to host my application?  Thanks!

**Reply by FatumaA** [reply]
Hey,
Welcome.
For Flutter yes you have to host it elsewhere. Supabase does not host sites. 
For the supabase backend: if using supabase cloud, you don't need to do any further setup. For self hosting, you can host elsewhere. This link has more info on that: https://supabase.com/docs/guides/self-hosting

**Comment by Lanr3waju** [comment]
🚀 Excited to join the Supabase community! 🎉 
This is my first time using Supabase, and I'm building a Web-Based Construction Management System that will revolutionize how we handle construction projects! 🏗️ It's going to be a centralized inventory hub for all the essential documents, project logs, and so much more! 📑🔧 And guess what? This ambitious project is my Final Year Project in college, so you can imagine just how much I'm invested in its success! 💪
Excited to be here and happy to contribute!

**Reply by FatumaA** [reply]
Welcome! 💚
That sounds like a really interesting project. I'd love to look at the code if it's public. 

I'm sure you are gonna kill it 💪🏽

**Comment by finsoncoutinho** [comment]
👋 Hey Everyone, my name is Finson Coutinho and I'm a Frontend developer👨‍💻. I'm using Supabase for my React project. Just loved the API documentation and how easy it is to connect it to a React app 🫶✨.

Also, I want to start my open-source contribution journey by contributing to Supabase. I want to work on this issue https://github.com/supabase/supabase/issues/16243. 

I would greatly appreciate it if someone could offer guidance to help me get started.

**Reply by FatumaA** [reply]
Hi, 
This is late but if you want to contribute simply say so on an issue that no one is already on. Then go ahead and make your contribution. 

You can find more info at: https://github.com/supabase/.github/blob/f0298106d7020f8ef077bbb93ed58b3db0069a30/CONTRIBUTING.md

**Comment by ErnestCrypto** [comment]
Hello my name is Ernest. I've been a supabase fan for awhile now and I'm very excited to be part of this community !!!!

**Reply by FatumaA** [reply]
Welcoming you in earnest, Ernest 😄🤭
Hopefully you joined the discord? If not here is the link: https://discord.com/invite/nZM6AJwS

**Comment by Nishitbaria** [comment]
Hello everyone, my name is Nishit Baria. I’m interested in exploring Supabase, a platform I’ve recently heard about quite a few times

**Reply by FatumaA** [reply]
Hi Nishitbaria,
Welcome 🙂

You can start with the docs => 
https://supabase.com/docs

Also, if you haven't yet, join the discord => 
https://discord.com/invite/bJFJMmUD

**Comment by mequann** [comment]
my middle ware is not working properly and un intended logout when try to navigate page to page and how can i fix it?

**Reply by FatumaA** [reply]
Hi Mequann 👋🏼,
What stack and tools are you using?

Also, it would be better to ask in the discord:
https://discord.gg/dyj6Wdad

**Comment by mequann** [comment]
I have used MERN Stack And vscode,github

**Reply by FatumaA** [reply]
I think it's better you join the discord and ask there. Please be detailed when you do

**Comment by msyamsul725** [comment]
Hello, I noticed that my Supabase database contains tables named spatial_ref_sys, geometry_columns, and geography_columns. I want to confirm if these tables are automatically created when enabling a GIS-related package or extension (like PostGIS) in Supabase.

I am concerned about whether these tables are part of the standard setup or if they could have been added by some other means. Could you please clarify their origin and whether they are safe to keep?

Thank you

**Comment by mequann** [comment]
I will look into this and get back to you.

**Comment by knowself** [comment]
Is there a valid discord link ?

http://discord.supabase.com is NOT it.

**Reply by FatumaA** [reply]
> Is there a valid discord link ?
> 
> http://discord.supabase.com is NOT it.

Hi!
Yes - https://discord.gg/h82DeF7J
And welcome to the community 😊

**Reply by knowself** [reply]
@FatumaA 

Thanks for the link, but this link did not work ?

![image](https://github.com/user-attachments/assets/8d135112-0d07-401a-9328-6db721133c2a)

**Reply by knowself** [reply]
@FatumaA 

Let me ask you about an issue.

`PS C:\Users\Owner\Documents\- - - - - - - Directories\HomeServicesRepairs.com\website\dev> supabase start
failed to load .env: unexpected character "»" in variable name near "\ufeffNEXT_PUBLIC_SUPABASE_URL=http://127.0.0.1:54321\nNEXT_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24iLCJleHAiOjE5ODM4MTI5OTZ9.CRXP1A7WOeoJeXxjNni43kdQwgnWNReilDMblYTn_I0"
`
I'm on windows 10 and Supabase is installed correctly in Docker for Desktop. But this issue of a the \ufeff (Byte Order Mark, BOM) error with supabase start. I'm sure that .env file is either empty or does not have that byte in it.

Sometimes i have the proper information in there ...

`NEXT_PUBLIC_SUPABASE_URL=http://localhost:8000
NEXT_PUBLIC_SUPABASE_ANON_KEY=your-actual-anon-key`

with my actual anon-key, but the first byte is NOT a BOM.

I just found out after working my brains out with GROK that the following would work.

Executed in Powershell, without elevated privileges.

$env:NEXT_PUBLIC_SUPABASE_URL = "http://localhost:8000"
$env:NEXT_PUBLIC_SUPABASE_ANON_KEY = "your-actual-anon-key"
supabase start --debug

... Bypassing the F'ING .env ????

**Reply by Marshall-Tech-Officiel** [reply]
Subject: Is it possible to recover a deleted database? (Urgent Help Needed)

Hi everyone,

I accidentally deleted the database of my friend's application, and unfortunately, it contained some very important and interesting data. I'm reaching out to ask if there's any possible way to recover a deleted database — maybe through Git history, backup mechanisms, or any other method that might help.

The database was not version-controlled, but I'm hoping someone here might know a workaround or tool that could assist in retrieving at least part of the lost data.

Any help or guidance would be really appreciated!

Thank you in advance.

**Reply by FatumaA** [reply]
@knowself sorry must have missed your comment. I dnt get what you were asking.. pls try and ask in the community - https://discord.gg/FGTRN29E

**Reply by FatumaA** [reply]
> Subject: Is it possible to recover a deleted database? (Urgent Help Needed)
> 
> Hi everyone,
> 
> I accidentally deleted the database of my friend's application, and unfortunately, it contained some very important and interesting data. I'm reaching out to ask if there's any possible way to recover a deleted database — maybe through Git history, backup mechanisms, or any other method that might help.
> 
> The database was not version-controlled, but I'm hoping someone here might know a workaround or tool that could assist in retrieving at least part of the lost data.
> 
> Any help or guidance would be really appreciated!
> 
> Thank you in advance.

Marshall, I'm sorry about that. Must be stressful. Paid projects have backups but for free ones its a bit more complicated.

This might be hard to recover from tbh but please contact support - https://supabase.com/support

I wish you the best!

**Comment by bijohn2** [comment]
Hi everyone, I hope all is well. Having joined this community yesterday due to issues with integration with Supabase and hoping to find a solution here (the sooner, the better), I initially posted in the community because I didn't see this page to introduce myself. I only returned to check after resting from the fatigue of three weeks of continuous effort. That said, I hope my connection here will foster more friendship, growth, and meaningful collaborations with everyone I meet. Lastly, I am a humble beginner in the tech world, coming from a health background and a Wordsmith.

**Reply by FatumaA** [reply]
> Hi everyone, I hope all is well. Having joined this community yesterday due to issues with integration with Supabase and hoping to find a solution here (the sooner, the better), I initially posted in the community because I didn't see this page to introduce myself. I only returned to check after resting from the fatigue of three weeks of continuous effort. That said, I hope my connection here will foster more friendship, growth, and meaningful collaborations with everyone I meet. Lastly, I am a humble beginner in the tech world, coming from a health background and a Wordsmith.

Hi bijohn2! 

Welcome to the supabase community 😊.

I'm sorry about the issues you were having with the supabase integration... did you post in the discord community? And is it now resolved?

**Reply by kimsoo527** [reply]
No

**Reply by kimsoo527** [reply]
#whats my link

**Comment by bijohn2** [comment]
Thank you for reaching out.  Issues still unresolved and yet to post on
discord.

**Comment by bijohn2** [comment]
No. What is this?

**Comment by FatumaA** [comment]
@bijohn2,
Here is a link to join the discord: https://discord.gg/XXtQVZ2D

You can post the issues you've been having in the help and questions channel once you've joined. 

That way there is more detail and you can get help.

**Comment by bijohn2** [comment]
Thanks, I'll check it out.

**Comment by tracklazy** [comment]
FUCK 2 DAYS NO ONE ANSWER ???? Subject: URGENT: My Project Is Blocked After Clicking Backup #37085

SUPABASE EMAIL : [garifiltet@gmail.com](mailto:garifiltet@gmail.com)

Dear Supabase Support Team,

I am reaching out urgently regarding a serious issue with my project.

After I clicked the “Backup” button in the dashboard, my entire Supabase project became blocked and is now completely inaccessible. This unexpected downtime has severely impacted my work.

I am currently on a Pro plan, and I expect reliable service and responsive support. I’ve already submitted a support request, but I have not received any resolution or update.

Please investigate and restore access to my project immediately. If additional action is needed on my side, kindly provide clear instructions.

This issue is extremely urgent.

Thank you for your prompt support.

Best regards,
Hamza

![81e44de33af22cb67742ea937a8892d](https://github.com/user-attachments/assets/353362ca-2b05-458c-8685-a94eecd533c9)
![a3746e90842aa5ca5b82dec9af1303f](https://github.com/user-attachments/assets/097e55a6-4502-4fbf-9d4b-1ec7472f4bb6)

**Reply by FatumaA** [reply]
Hi Hamza,
Apologies for this. I will ping the team for you ASAP.

**Comment by bijohn2** [comment]
Why copy me in on this case? I don't work for supabase and I don't know
this person. My case received no answer from support or in the community 😕

**Reply by FatumaA** [reply]
Hi,

What was your case please? And did you ask for help in the discord?

**Comment by bijohn2** [comment]
Here is the number:
Thanks for reaching out to us. We have received your support request, and
your ticket id is SU-221327. This is for school work, and I am very behind
with it. Thank you again. PS: I posted it on the school account.

**Reply by FatumaA** [reply]
@jenniekibiri Hi, Pls help follow up on this

**Reply by jenniekibiri** [reply]
Checking...

**Reply by jenniekibiri** [reply]
@FatumaA  Responded to both.

**Reply by FatumaA** [reply]
Thanks!

**Comment by udneymanxe** [comment]
🚨 URGENT: My @supabase account got banned after weeks of using it for a production web app (Lovable Shipped Program). No response from support despite multiple emails. Users are affected. Please help!

**Reply by FatumaA** [reply]
Hi,
I'm sorry about your experience. Is there any reason it got banned?

You could reach out to the supabase team via twitter by tagging the offiicial account (@supabase), they are pretty responsive over there

**Reply by freshmann7** [reply]
How do I connect loveable to supabase to create my app?

**Reply by FatumaA** [reply]
> How do I connect loveable to supabase to create my app?

https://docs.lovable.dev/integrations/supabase

**Comment by bijohn2** [comment]
...hope your issue gets resolved.  My is still in the wind.

**Reply by FatumaA** [reply]
Hi, 
Apologies for that. 

Here is a guide on integrating lovable with supabase - https://docs.lovable.dev/integrations/supabase

**Comment by AGBE12** [comment]
Bonjour,

Je travaillais avec l’équipe Supabase sur l’intégration d’un système de Token dans mon projet.
Nous échangions normalement, mais depuis quelques jours, chaque fois que j’envoie un message via le chat de support, je reçois uniquement la réponse :

“Sorry, I’m having trouble responding right now.”

Je souhaiterais savoir si c’est un problème technique ou une limitation liée à mon plan actuel.
Mon objectif est de continuer la discussion pour finaliser la partie Token et intégrer certaines améliorations proposées par Supabase.

Ticket ID : SU-233025

Merci d’avance pour votre aide et vos conseils pour débloquer la situation.

**Reply by FatumaA** [reply]
Bonjour,
Est-ce que ce problème a été résolu ? Je vais alerter l’équipe à ce sujet.

Merci.

---
Hi, 
Has this been resolved? I will alert the team about the issue. 

Thank you

**Reply by FatumaA** [reply]
@jenniekibiri  also here

**Reply by jenniekibiri** [reply]
Checking...

**Comment by richardkyere2016-blip** [comment]
Hi everyone!
 I’m Richard Kyeré from Ghana.
I’m new to Supabase and just getting started with an idea called Edunovate Ghana — a platform to support educators with smart tools, feedback systems, and AI assistance. My goal is to build something that helps teachers and students collaborate better, even in areas with limited internet access.
I’m not a full-time developer, but I’m learning as I go and really excited about how Supabase can help me manage data securely and build scalable features. Looking forward to learning from this amazing community and sharing my progress

**Reply by FatumaA** [reply]
Hi,
Welcome, you should check out the Discord - https://discord.gg/MBt7bp2B

**Comment by Colomandev** [comment]
Do anyone know of a course for Supabase integration and authentication.... I am trying to get some tutoring on supabase. Thanks

**Reply by FatumaA** [reply]
Hi, 
There are lots of resources for this, including the docs - https://supabase.com/docs/guides/auth and official vids: https://www.youtube.com/watch?v=v3Exg5YpJvE&list=PL5S4mPUpp4OvE6MLbO1xAMvKLKT1sFsHF

There are also some community resources on the same:

- An amazing vid on SSR by activenode - https://www.youtube.com/watch?v=XIj7nmIYtbo
- This channel is also a great resource - https://www.youtube.com/@codewithguillaume/videos

I also create educational content on Supabase, focusing primarily on authentication. You can find my videos here - https://www.youtube.com/@HijabiCoder/videos and blogs - https://www.freecodecamp.org/news/author/HijabiCoder/

Hope that helps!

**Reply by orcn-dev** [reply]
> Do anyone know of a course for Supabase integration and authentication.... I am trying to get some tutoring on supabase. Thanks

You don’t necessarily need a dedicated course — the Supabase docs are actually pretty solid, especially the [Auth guide](https://supabase.com/docs/guides/auth)
 and 
 [Examples repo](https://github.com/supabase/supabase/tree/master/examples)
.

If you prefer video, Traversy Media on YouTube has a “Supabase Crash Course” which covers integration + authentication basics in a very practical way.

My own tip: start with email/password auth, then add providers (Google/GitHub) once you’re comfortable. That path helped me a lot. 🚀

**Reply by aantti** [reply]
David @ activenode also wrote a good comprehensive book covering these topics :) The auth part and structuring database properly is quite a significant part of it. No affiliation, just been reading it lately to learn more, and thought I'd share. ("Building Production-Grade Web Applications with Supabase")

**Comment by uchitwai** [comment]
Hello World

**Comment by MohammedHani98** [comment]
Hi,
My project is showing "pausing" and freezing at this endless loading though it does not reach the 90 days threshold.

Ticket ID: SU-256912
Project ID: nxpqmtrydoutfvzcpoij

Thanks,

<img width="1630" height="699" alt="image" src="https://github.com/user-attachments/assets/e717f6e8-4c48-4204-b941-484a8309f804" />

**Reply by FatumaA** [reply]
Hi, 
Did this resolve?

**Reply by MohammedHani98** [reply]
Hi Fatuma,

Unfortunately the problem still persists.

**Reply by FatumaA** [reply]
Will pass along and let you know

**Reply by MohammedHani98** [reply]
Hi @FatumaA, Any news regarding this matter?

**Reply by FatumaA** [reply]
Hi there,
Awaiting feedback but the matter is being looked into

**Reply by MohammedHani98** [reply]
Thank you Fatuma the problem is now resolved 👍

**Reply by saltcod** [reply]
Just seeing this @MohammedHani98, apologies. In the future, a support ticket would be your quickest way to get this resolved.

**Reply by MohammedHani98** [reply]
Hello @saltcod, No worries. Thanks 👍.

**Comment by temitopeakin1** [comment]
hey everyone, i am excited to be here, I've been using supabase for the past three years . I am much more excited to contribute to this project

**Reply by FatumaA** [reply]
Hi Temi,
Welcome! Please feel free to contribute in any capacity that works best for you. Here are some ways to get started:

1. Code - You can help triage and reproduce issues, and you can help fix some of the open issues
2. Docs - You can update the docs that need it, or add where clarification is needed. You can also add examples
3. Community - You can help answer questions on Discord, Reddit, or Twitter. You can also help raise awareness of the product. You can help host launch week supabase meetups

**Reply by rishabh2002chauhan** [reply]
discord link?
previous ones are not working

**Reply by aantti** [reply]
I think it's this https://discord.supabase.com/

**Reply by cookiemariepearson-coder** [reply]
> hey everyone, i am excited to be here, I've been using supabase for the past three years . I am much more excited to contribute to this project

what projects have you completed and which system did you use, if you dont mind me asking?

**Comment by ThatBobo** [comment]
Hello, I'm @ThatBobo, I don't know everything of supabase, but I will get an issue for a bug soon

**Reply by ThatBobo** [reply]
> Hello, I'm @ThatBobo, I don't know everything of supabase, but I will get an issue for a bug soon

Ok, it's #45532

**Comment by snautomationbd** [comment]
I recently upgraded my project to the Pro plan at $25/month because my Free Plan egress usage had reached approximately 98%, and I was concerned that my project might be interrupted after reaching the limit.

I really like Supabase and would like to continue using it long-term. However, $25 per month is quite expensive for my current project and is difficult for me to maintain every month.

My actual requirements are much smaller than the resources included in the current Pro plan. For example, I do not need 250 GB of egress or 100 GB of database storage.

A plan with approximately the following limits would be more than enough for my needs:

• Around 50 GB monthly egress
• Around 50 GB database storage
• Around 1,000,000 Edge Function invocations per month would be sufficient
• I do not need the full resource limits included in the $25 Pro plan

My sustainable monthly budget is around $5–$10 USD. Ideally, a $5 plan would be very helpful, but I could also manage around $10 per month if necessary.

I would be very interested in something similar to:

• A smaller plan around $5/month with reduced resources, or
• A plan around $10/month providing roughly half or less of the resources included in the current Pro plan.

I understand that these may not be part of your standard pricing tiers, but I wanted to ask whether Supabase offers any custom plan, lower-cost plan, startup/individual developer option, discounted plan, or other arrangement for users with relatively modest resource requirements.

I have already subscribed to Pro for this billing cycle because the egress limit was becoming critical. Another reason I upgraded was that I previously tried to contact support while using the Free Plan, and I understood that direct support availability is more limited on the Free tier.

I am not looking for a temporary solution—I expect to use Supabase every month. I simply need a plan that is financially sustainable for a smaller project.

I believe lower-cost tiers such as $5 and $10 could also be very helpful for independent developers and small projects that need more resources than the Free Plan but do not require the full Pro allocation.

Could you please let me know if there is any lower-cost or customized option available for my project?

I would be very grateful if you could consider my request.

Thank you for your time and support.

Best regards,
SN Automation

**Reply by k1ng-arthur** [reply]
Self hosting on a VPS is always an option

**Comment by kevindmoore** [comment]
That is what I did. Using AI, I was able to create 3 different supabase
instances. It's a bit slower than hosted, but works well.

Thanks,
Kevin Moore

**Comment by cookiemariepearson-coder** [comment]
Hello,
Im new to the group, thanks for having me. I just want to know some projects you built and how long did it take?

**Reply by kevindmoore** [reply]
I will say that it didn't take long to write projects. I use Flutter and created a package to use supabase

**Comment by haha01021** [comment]
你们为什么不使用微信支付或支付宝呢？作为中国用户使用银行卡进行境外支付很麻烦。

**Comment by cookiemariepearson-coder** [comment]
Ok no problem I will add that soon.
